EOSatelliteImagery/schema.json

```diff
@@ -0,0 +1,102 @@
+{
+   "$schema":"http://json-schema.org/schema#",
+   "$schemaVersion":"0.0.1",
+   "modelTags":"",
+   "derivedFrom":"",
+   "license":"",
+   "$id":"https://smart-data-models.github.io/dataModel.SatelliteImagery/EOSatelliteImagery/schema.json",
+   "title":"Satellite Imagery - EO Unified Satellite Imagery",
+   "description":"This schema defines a unified entity within the SatelliteImagery domain, consolidating attributes from all existing entities related to satellite imagery, designed to offer a holistic view in the EO proccess",
+   "type":"object",
+   "allOf":[
+      {
+         "$ref":"https://smart-data-models.github.io/data-models/common-schema.json#/definitions/GSMA-Commons"
+      },
+      {
+         "$ref":"https://smart-data-models.github.io/data-models/common-schema.json#/definitions/Location-Commons"
+      },
+      {
+         "properties":{
+            "type":{
+               "type":"string",
+               "description":"Property. NGSI Entity type. It must be equal to EOSatelliteImagery",
+               "enum":[
+                  "EOSatelliteImagery"
+               ]
+            },
+            "productBbox":{
+               "$ref":"https://smart-data-models.github.io/data-models/common-schema.json#/definitions/Location-Commons/properties/location"
+            },
+            "processedBbox":{
+               "$ref":"https://smart-data-models.github.io/data-models/common-schema.json#/definitions/Location-Commons/properties/location"
+            },
+            "analysisType":{
+               "type":"string",
+               "description":"Property. Entity's type of analysis applied"
+            },
+            "provider":{
+               "type":"string",
+               "description":"Property.  Model:'https://schema.org/Text'. The provider of the algorithm"
+            },
+            "contentInformation":{

```

Comment:
you have to include also the descriptions in the subproperties. 
layer_name, layer_categorization, values_explanation

Reply:
The description for the "items" has descriptions for all the different properties (layer_name, layer_categorization, values_explanation). Do you prefer to split it up with different descriptions for each one separately?